tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/Summary.java

```diff
@@ -79,4 +83,50 @@ public Response doVettingSummary(
             return Response.status(500, "An exception occurred").entity(ioe).build();
         }
     }
+
+    @GET
+    @Path("/dashboard/{locale}/{level}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+        summary = "Fetch the Dashboard for a locale",
+        description = "Given a locale, get the summary information, aka Dashboard"
+    )
```

Comment:
I like the name Dashboard better than Summary

Reply:
@btangmu this API class has both… i'm glad to rename something if it really helps